test/integration/submoduleAdd/setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -e
+
+cd $1
+
+export GIT_COMMITTER_DATE="Mon 20 Aug 2018 20:19:19 BST"
```

Comment:
added these to make our commit SHAs the same, though I don't think it actually ended up being the reason for the failing tests. Still probably a good idea to have. Will need to standardise this